tests/vm_test.go

```diff
@@ -248,21 +249,7 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		)
 
 		It("[test_id:3161]should carry annotations to VMI", func() {
-			annotations := map[string]string{
-				"testannotation": "test",
-			}
-
-			vm := createVM(virtClient, libvmifact.NewCirros())
-
-			err = tests.RetryWithMetadataIfModified(vm.ObjectMeta, func(meta k8smetav1.ObjectMeta) error {
```

Comment:
This requires a good reasoning on why it is ok to remove and what replaces it.
It fits a different commit then the one that replaces it the update-retry with a patch.

Each should explain well in the commit message what was the alternative to the update and why it works. There is a need to convince the reviewer and the future reader of this change that it correct and the thoughts around it.

Reply:
Thanks. Done here: https://github.com/kubevirt/kubevirt/pull/11948.